### DISCUSSION TITLE
API is not returning PRO users

### DISCUSSION THREAD

**Discussion by lipecss** [community/community#24801] [Apps, API and Webhooks]
<p>I was using API to get users, at DOCS tell that we can identify who is PRO Or not, but im search people who are PRO e that field  <strong>plan</strong>  is not come</p>

**Comment by seveas** [comment, marked as answer]
<p>I’m afraid you may have misread the documentation. The documentation for the /users endpoint does not mention plans at all, and the documention for /user/:username states that only the authenticated user or an app with the plan permission can see it.</p>
